Stream_support/include/CGAL/IO/GOCAD.h

```diff
@@ -158,6 +158,8 @@ bool read_GOCAD(std::istream& is,
 }
 /// \endcond
 
+/// \endcond
+
```

Comment:
Looks like a double fix attempt, see also https://github.com/CGAL/cgal/pull/6199/files and as seen in https://cgal.geometryfactory.com/CGAL/Manual_doxygen_test/CGAL-5.4-Ic-137/logs_master/Stream_support.log

Reply:
I paid attention to that in Ic-136 but forgot after the merge. Thanks for the reminder!